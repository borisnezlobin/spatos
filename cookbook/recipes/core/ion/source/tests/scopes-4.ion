fn demo
  echo ${super::foo}
  drop foo

  fn bar
    super::demo
  end
  bar
end

let foo = bar
echo $foo
demo
