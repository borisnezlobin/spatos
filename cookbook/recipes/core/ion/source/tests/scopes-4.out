bar
bar
ion: undefined variable: foo
ion: pipeline execution error: command not found: super::demo
